Tools/buildstream/elements/sdk/sysprof-capture.bst

```diff
@@ -0,0 +1,28 @@
+kind: meson
+
+build-depends:
+- freedesktop-sdk.bst:public-stacks/buildsystem-meson.bst
+
+depends:
+- sdk/libdex.bst
+
+variables:
+  meson-local: >-
+    -Dgtk=false
+    -Dsysprofd=host
+    -Dhelp=false
+    -Dlibsysprof=false
+    -Dtools=false
+    -Dtests=false
+    -Dexamples=false
+
+sources:
+- kind: tar
+  url: gnome_downloads:sysprof/46/sysprof-46.0.tar.xz
```

Comment:
AFAIK Niko recently landed a patch in sysprof master that we might need. Should it be included?

Reply:
Niko's patch is for `sysprof-cli`, which is not being built in this Buildstream recipe (because of `-Dtools=false` in the Meson options().